Update remove-former-employee-step-4.md (#11345)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/remove-former-employee-step-4.md

5. Select **Save**.
6. Remember, don't delete the former employee's account.

> [!IMPORTANT]

> > In Scenarios where in the mailbox is Inactive for complaince reasons, the mail forwarding or converting it to a shared mailbox would not work.
> 

Reviewer: ```suggestion
> In scenarios where the mailbox is inactive due to compliance reasons, mail forwarding or converting the mailbox to a shared mailbox would not work.
```
